demisto_sdk/commands/unify/tests/unifier_test.py

```diff
@@ -253,6 +253,29 @@ def test_insert_image_to_yml():
         assert yml_unified == yml_unified_test
 
 
+def test_insert_image_to_yml_without_image(tmp_path):
+    """
+    Given:
+     - Integration without image png file
+
+    When:
+     - Inserting image to unified YAML
+
+    Then:
+     - Ensure the insertion does not crash
+     - Verify no image path is returned
+    """
+    integration_dir = tmp_path / 'Integrations'
+    integration_dir.mkdir()
+    integration_yml = integration_dir / 'SomeIntegration.yml'
+    integration_obj = {'id': 'SomeIntegration'}
+    yaml.dump(integration_obj, integration_yml.open('w'), default_flow_style=False)
```

Comment:
you should delete the file during teardown

Reply:
it is handled by `pytest`